.github/workflows/ui.yml

```diff
@@ -57,3 +57,24 @@ jobs:
         run: |
           . venv/bin/activate
           pnpm run test:ct
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - name: install gradio
+        run: python -m pip install gradio==$(cat gradio/version.txt)
+      - name: generate theme.css
+        run: python scripts/generate_theme.py
+      - name: install dependencies
+        uses: "./.github/actions/install-frontend-deps"
+        with:
+          always-install-pnpm: true
```

Comment:
Since we are running both the python and npm installs we can just use the `install-all-deps` action instead it icludes a bunch of caching and stuff which can speed things up a lot. You can see it used here: https://github.com/gradio-app/gradio/blob/main/.github/workflows/publish-npm.yml#L20-L26

This also skips building the gradio app (frontend) since we don't need it here you could do the same. You wouldn't need to pass the two tokens, thats just for the npm publish auth stuff.